docs/UserGuide.md

```diff
@@ -166,6 +166,22 @@ Example usage of delete command in BayMeds:
 
 ![Example of delete command](images/ui/Ui-delete.png)
 
+### Listing Untaken Medication: `list today`
+
+Lists all remaining medications to be taken for the day.
+
+Format:
+```
+list today
+```
+
+Examples:
+* `list today`.
+
+Example usage of List Today command in BayMeds:
+
+![Example of list today command](images/ui/Ui-list_today.png)
+
```

Comment:
ps can help change to `<img src.....> `format (smth i just made in my PR; to standardise the img size).

and if doing List Today. Should change the alternative text to "List Today" instead of "list today"

Reply:
SRY BRO